src/types.ts

```diff
@@ -0,0 +1,3 @@
+import parsePokemon from "./hooks/utils/parsePokemon";
+
+export type Pokemon = ReturnType<typeof parsePokemon>;
```

Comment:
TBH, this is risky practice. In bigger project, it would be safer to write this type manually, and mark `parsePokemon` as `function parsePokemon(pokemon: GqlPokemon): Pokemon`
Right now, if some broken type (like `any`) will be returned by your function, this will propagate into your whole codebase.

Reply:
fixed in search bar PR